src/Components/Nav/Nav_Components/Category.js

```diff
@@ -1,10 +1,13 @@
-import React, { Component, useState } from "react";
-import styled, { keyframes } from "styled-components";
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
 
 function Category({ title, list }) {
   return (
```

Comment:
```suggestion
function Category({ title, list }) {
  const [foo, bar, baz] = list;
  return (
```
사용된 list 내의 아이템들 구조분해할당 해주세요!

Reply:
수정했습니다!